src/VisionModel/googlenet.jl

```diff
@@ -0,0 +1,42 @@
+export googlenet
+
+function conv_block(inplanes, outplanes, stride, pad, kernel_size)
+  conv_layer = Chain(Conv(kernel_size, inplanes => outplanes, stride = stride, pad = pad),
+                     BatchNorm(outplanes, relu))
+  return conv_layer
+end
+
+function inception_block(inplanes, out_1x1, red_3x3, out_3x3, red_5x5, out_5x5 out_1x1pool)
```

Comment:
Can we rename the arguments according to the columns of Table 1 in the paper?

(Also there is a comma missing in the argument list)

Reply:
I have changed last argument to ``pool_proj``